proto/buf/alpha/registry/v1alpha1/plugin.proto

```diff
@@ -48,6 +48,8 @@ message PluginVersion {
   string plugin_name = 3;
   // The owner of the plugin to which this version relates.
   string plugin_owner = 4;
+  // The full docker image digest of this plugin version.
+  string digest = 5;
```

Comment:
I have a minor level of concern that just calling this `digest` overlaps with our concept of digest for modules, perhaps we should clarify this is a docker digest, I'm not sure.

Reply:
Would either of `ImageDigest` or ManifestDigest` overlap less with our existing nomenclature?

I think I prefer `ManifestDigest` because it's not a term we've used (afaics)

Reply:
Updated to `image_digest`. I think most folks are used to the term "image" when it comes to docker images so it's slightly more familiar as compared to `manifest_digest`.